core/src/test/java/nl/nn/adapterframework/http/HttpMessageEntityTest.java

```diff
@@ -27,12 +31,12 @@ public class HttpMessageEntityTest {
 	private static Message NON_REPEATABLE_MESSAGE;
 	private static Message REPEATABLE_TS_MESSAGE;
 
-	@BeforeClass
+	@BeforeAll
```

Comment:
I know you did not write this test, but I don't like the way this test is set up with `@BeforeAll` instead of regular non-static `@Before`. There are things inside message class that change its state when accessing streams and that means that potentially, each test is executed with a slightly different state of the `Message` instance(s).